src/helics/application_api/HelicsPrimaryTypes.hpp

```diff
@@ -34,14 +34,14 @@ using defV = std::variant<double,
                           NamedPoint>;
 
 /**enumeration of the order inside the variant so the Which function returns match the enumeration*/
```

Comment:
Unrelated to the warnings, but I'm a bit confused what this comment is saying.

Is it something like "enumeration of the order inside the variant so the order of the function returns match the enumeration"?

Reply:
yes